odoo/addons/base/tests/test_image.py

```diff
@@ -177,6 +176,20 @@ def test_13_image_process_quality(self):
         res = tools.image_process(self.img_1920x1080_jpeg)
         self.assertLessEqual(len(res), len(self.img_1920x1080_jpeg))
 
+        # CASE: JPEG optimize + bigger size => original
+        image = Image.new('RGB', (1920, 1080), color=self.bg_color)
+        offset = (image.size[0] - image.size[1]) / 2
```

Comment:
I'm not sure to understand what are `image.size[0]` and `image.size[1]`. Can you explain ?

Reply:
They are the images width and height. It is just used to position an ellipse within the image. I can use a hard-coded number instead if it reduces confusion.